core/src/wsv.rs

```diff
@@ -101,12 +93,12 @@ pub struct WorldStateView {
     blocks: Arc<Chain>,
     /// Hashes of transactions
     pub transactions: DashSet<HashOf<VersionedSignedTransaction>>,
-    /// Metrics for prometheus endpoint.
-    pub metrics: Arc<Metrics>,
-    /// Notifies subscribers when new block is applied
-    new_block_notifier: Arc<NewBlockNotificationSender>,
-    /// Transmitter to broadcast [`WorldStateView`]-related events.
-    events_sender: EventsSender,
+    /// Temporary buffer containing events generated during `WorldStateView::apply`.
```

Comment:
I feel saying that it's temporary is a bit confusing, and stating when it's updated would be beneficial.
```suggestion
    /// Buffer containing events generated during `WorldStateView::apply`. Renewed on every block commit.
```

Reply:
Good suggestion.